server/enterprise.go

```diff
@@ -25,6 +26,7 @@ func (s *Server) triggerEnterpriseTests(pr *model.PullRequest) {
 
 	triggerInfo, err := s.getPRInfo(ctx, pr)
 	if err != nil {
+		mlog.Debug("error trying to get pr info", mlog.Err(err))
```

Comment:
mlog.Warn would be preferred.

Reply:
:+1: I didn't want to clutter the logs with it but you're right better warn